src/App.js

```diff
@@ -86,12 +88,33 @@ function Nav() {
   const pfp = auth.currentUser.photoURL;
   const name = auth.currentUser.displayName;
 
-  return (
-    <nav>
-      <img alt="pfp.png" src={pfp}></img>
-      <h3>Logged in as: {name}</h3>
-    </nav>
-  );
+  if(maintainers.includes(vars.user)) {
+    return (
+      <nav>
+        <img alt="pfp.png" src={pfp}></img>
+        <h3>Logged in as: {name} (@is-a-dev/maintainers)</h3>
+      </nav>
+    );
+  } else if(helpers.includes(vars.user)){
+    return (
+      <nav>
+        <img alt="pfp.png" src={pfp}></img>
+        <h3>Logged in as: {name} (@is-a-dev/helpers)</h3>
+      </nav>
+    );
+  } else if(vars.user === "phenax"){
```

Comment:
I don't think we need an extra condition for this here. Also since I'm in the maintainers array, this condition will occur anyway.

Reply:
True